chain/consensus/mir/validator/membership.go

```diff
@@ -32,3 +36,27 @@ type EnvMembership string
 func (e EnvMembership) GetValidatorSet() (*Set, error) {
 	return NewValidatorSetFromEnv(string(e))
 }
+
+// -----
+
+const JSONRPCServerURL = "http://127.0.0.1:3030"
+
+type ActorMembership struct {
+	client rpc.JSONRPCRequestSender
+}
+
+func NewActorMembershipClient(client rpc.JSONRPCRequestSender) *ActorMembership {
+	return &ActorMembership{
+		client: client,
+	}
+}
+
+// GetValidatorSet gets the membership config from the actor state.
+func (c *ActorMembership) GetValidatorSet() (*Set, error) {
+	var set Set
+	err := c.client.SendRequest("Filecoin.GetValidatorSet", nil, &set)
```

Comment:
This is wrong, the method name is the following: https://github.com/consensus-shipyard/ipc-agent/blob/5815f5955c1eaaf4bf07637e460cfbcee1f2bdf4/src/config/server.rs#L20

Reply:
Fixed